frontend/src/libraries/explorer-wamp/details.ts

```diff
@@ -2,7 +2,7 @@ import { ExplorerApi } from ".";
 
 export interface Details {
   accountsCount: number;
-  totalNodesCount: number;
+  ValidatorCount: number;
```

Comment:
JS code style suggests using cameCase starting with the lowercase letter for variable names and fields.